core/views/ticket.py

```diff
@@ -17,3 +18,38 @@ def ticket(request, ticket_id: int = None):
         else:
             context["ticket"] = ticket
         return render(request, "core/ticket.html", context)
+
+
+def get_boarding_position(current_bookings: int):
+    group = chr((current_bookings // 10) + 65)
+    number = (current_bookings % 10) + 1
+    return group, number
+
+
+def book_ticket(request):
+    if request.method == "POST":
+        try:
+            flight = Flight.objects.prefetch_related("ticket_set").get(id=request.POST.get("flight_id"))
+        except Flight.DoesNotExist:
+            return redirect("/book/") #TODO: Add error message
+        else:
+            current_bookings = flight.ticket_set.count()
+            checkbox = True if request.POST.get("tsa_precheck") is not None else False
+            if current_bookings < flight.capacity:
+                group, number = get_boarding_position(current_bookings)
+                ticket = Ticket.objects.create(
+                    flight=flight,
+                    first_name=request.POST.get("customer_name"),
+                    costume=request.POST.get("customer_costume"),
+                    tsa_precheck=checkbox,
+                    boarding_group=group,
+                    boarding_position=number,
+                )
+                ticket.save()
```

Comment:
Does this have the possibility of assigning multiple people the same boarding number?

Reply:
yep! To be addressed in a subsequent PR :)